packages/teams-js/src/internal/utils.ts

```diff
@@ -400,3 +400,22 @@ export function ssrSafeWindow(): Window {
 export function inServerSideRenderingEnvironment(): boolean {
   return typeof window === 'undefined';
 }
+
+/**
+ * @param id The ID to validate against the UUID format
+ * @throws Error if ID is not a valid UUID
+ *
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export function validateUuid(id: string | undefined | null): void {
+  if (id === undefined || id === null) {
+    return;
+  }
```

Comment:
Is it correct that `undefined` and `null` are "valid" UUIDs? Aren't those invalid?

Reply:
Updated to make all these cases invalid.